docs/developers/node-tutorial.mdx

```diff
@@ -704,7 +704,7 @@ following command:
 <TabItem value="arabica" label="Arabica">
 
 ```sh
-celestia light start --core.ip <ip-address> --core.grpc.port <port> --keyring.accname <key_name> --gateway --gateway.addr 0.0.0.0 --gateway.port 26659
+celestia light start --p2p.network arabica-2 --core.ip <ip-address> --core.grpc.port <port> --keyring.accname <key_name> --gateway --gateway.addr 0.0.0.0 --gateway.port 26659
```

Comment:
thanks for the contribution, it will work with just `arabica` as it's an alias for arabica-2. Can you update to just `arabica` wherever you mention `arabica-2`